### ISSUE TITLE
setDateTimeInterpreter with custom views...

### ISSUE THREAD

**Issue by dirkvranckaert** [alamkanak/Android-Week-View#46]
It would be really awesome to have the possibility to set a DateTimeInterpreter (maybe called a DateTimeViewInterpreter) which could return the view to be displayed in the header.

I want that because I want something like the Google Calendar app above my day columns that show me "Wed 7", but on two lines and "7" a lot bigger than "Wed".

**Comment by dirkvranckaert** [comment]
I would also be able this way to highlight weekends, holidays or other special days for my app

**Comment by dirkvranckaert** [comment]
In fact the first column with the hours should have the width of the longest label/view. Then the day columns should have the width of the widest header label/view.

**Comment by dirkvranckaert** [comment]
Yes indeed that would be an excellent solution with the DateTimeViewInterpreter.

**Comment by user32338** [comment]
i need it toooooo
